tests/fcidump/input.dat

```diff
@@ -5,19 +5,77 @@ molecule ne {
 set {
   basis 6-311g
   scf_type pk
+  reference rhf
+  d_convergence 1e-8
+  e_convergence 1e-8
 }
 
 scf_e, scf_wfn = energy('SCF', return_wfn=True)
 fcidump(scf_wfn, fname='Ne.6311G.INTDUMP', oe_ints=['EIGENVALUES'])
-compare_fcidumps('Ne.6311G.INTDUMP.ref', 'Ne.6311G.INTDUMP', 'Ne, 6-311g, integrals in FCIDUMP format')  # TEST
+energies = compare_fcidump_headers('Ne.6311G.INTDUMP.ref', 'Ne.6311G.INTDUMP', 'Ne, 6-311g, integrals in FCIDUMP format') # TEST
+compare_values(get_variable("ONE-ELECTRON ENERGY"), energies['ONE-ELECTRON ENERGY'], 7, "Ne, 6-311g, RHF 1-electron energy") #TEST
```

Comment:
should `get_variable("ONE-ELECTRON ENERGY")` and `energies['ONE-ELECTRON ENERGY']` be reversed for `ref, computed`?

Reply:
`get_variable` comes from the computed wave function, `energies` from reading the integrals from file. So I think they are in the correct order already.